src/index.css

```diff
@@ -1,4 +1,294 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@700&family=Roboto+Slab&display=swap');
+
 * {
   box-sizing: border-box;
   padding: 0;
 }
+
+body {
+  background-color: #f5f6fa;
+  padding: 30px;
+}
+
+main {
+  background-color: #fafafa;
+  padding: 2.313rem 6.25rem 16.563rem 6.25rem;
+}
+
+.navContainer {
+  display: flex;
+  height: 5.938rem;
+  background-color: #fff;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  justify-content: space-between;
+}
+
+.navDiv {
+  display: flex;
+}
+
+.title {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.875rem;
+  color: #0290ff;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+}
+
+.navLinks > a {
+  text-decoration: none;
+  margin: 1.125rem 2.563rem 0.688rem 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+}
```

Comment:
- [ ] Can you please transform the navbar links to uppercase as given in the project guidelines for getting a replica of the Zeplin's design ? 😃📌

Your design | Given design 
---| ---|
![image](https://github.com/lorenzoworx/bookstore-react/assets/106119105/7716d13c-1684-4cb3-bcd7-664314cc4f7f) | ![image](https://github.com/lorenzoworx/bookstore-react/assets/106119105/3813b889-cb7c-4430-8985-2f5add91ad46)